lms/templates/stanford/_course_ga_tag.html

```diff
@@ -0,0 +1,11 @@
+% if course_ga_tag:
+    <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
Do we actually need to declare this?
Shouldn't the `ga` object already be present in the global namespace by this point?
(initial inspection indicates that it is).

Assuming that's always the case, we can drop this `IIFE` and have a `SCRIPT` as simple as

```javascript
ga('create', '${course_ga_tag}', 'auto');
ga('send', 'pageview');
```